modules/apps/transfer/ibc_module_test.go

```diff
@@ -319,3 +319,94 @@ func (suite *TransferTestSuite) TestOnChanUpgradeInit() {
 		})
 	}
 }
+
+func (suite *TransferTestSuite) TestOnChanUpgradeTry() {
+	var path *ibctesting.Path
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expError error
+	}{
+		{
+			"success",
+			func() {}, // successful happy path for a standalone transfer app is swapping out the underlying connection
+			nil,
+		},
+		{
+			"invalid upgrade connection",
```

Comment:
Same comment: maybe this test is not relevant here?

Reply:
Yep! If necessary we can remove post-audit, I think I'll keep it here for now regardless of it being more of a core test